### ISSUE TITLE
Specializing Word Embeddings for Similarity means method "Joint-Thesaurus" or "SG-Retrofit"?

### ISSUE THREAD

**Issue by rollinglyj** [huangmiaoxin/SpecializeWordEmbedding#1]
Not familiar with Torch, it seems not like "SG-Retrofit" ?

**Comment by huangmiaoxin** [comment]
sorry, saw the comment just now. 
yes this code use 'Joint-Thesaurus'.

**Comment by rollinglyj** [comment]
Did you ever implement  "SG-Retrofit"  ? Which seems works better in the paper?

**Comment by huangmiaoxin** [comment]
No, I just consider that 'SG-retrofit' which individually trains word vector with thesaurus followed by corpus is equal to fine-tune word2vec result by using a similarity thesaurus. But the similarity thesaurus I extracted is too small to get a great fine-tune result. So I think corpus and thesaurus training together is a trade-off solution.
